src/components/video-button-modal/video-button-modal.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+
+import Modal from '../modal/modal.jsx'
+import VideoPlayer from '../video-player/video-player.jsx'
+
+import defaultImage from '../../../assets/img/meine_sachen.png'
+import styles from '../menu-listing/menu-listing.css'
+
+const VideoButtonModal = ({ title, image, note, video }) => {
+  const [showModal, setShowModal] = useState(false)
+  return (
+    <>
+      <div
+        className={styles.projectWrapper}
+        role="button"
+        onClick={() => setShowModal(true)}
+      >
+        <span className={styles.title}>{title}</span>
+        <span className={styles.image}>
+          {image ? <img src={image} /> : <img src={defaultImage} />}
```

Comment:
```suggestion
          <img src={image || defaultImage} />
```

Reply:
I did test it in the browser, added some minor text changes and talk to Nadja about tiny styling changes.